js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/ParameterEdgeDirection.js

```diff
@@ -0,0 +1,45 @@
+import React, { useContext, useState } from "react";
+import { UrlParametersContext } from "./url-parameters-context";
+import Checkbox from "./components/pure-css/form/Checkbox.tsx";
+import ToolTip from '../../components/arango/tootip';
+import {
+  Box,
+  Flex,
+  Center } from '@chakra-ui/react';
+
+const ParameterEdgeDirection = () => {
+  const [urlParameters, setUrlParameters] = useContext(UrlParametersContext);
+  const [edgeDirection, setEdgeDirection] = useState(urlParameters.edgeDirection);
+
+  const getNewUrlParams = (urlParams, newEdgeDirection) => {
```

Comment:
this urlParams is unused

Reply:
Resolved within https://github.com/arangodb/arangodb/pull/18199/commits/e7d28fec4347f6203decf73a97e0bb7d3349768d